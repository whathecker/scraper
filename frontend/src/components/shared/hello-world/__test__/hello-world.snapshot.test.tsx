import * as React from 'react';
import HelloWorld from '../hello-world';
import { shallow } from 'enzyme';

describe('Snapshot testing of HelloWorld component', () => {
  test('renders correctly', () => {
    const tree = shallow(<HelloWorld />);
    expect(tree).toMatchSnapshot();
  });
});
