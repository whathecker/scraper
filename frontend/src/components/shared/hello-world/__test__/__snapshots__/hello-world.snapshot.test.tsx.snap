// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing of HelloWorld component renders correctly 1`] = `ShallowWrapper {}`;
